pkgs/by-name/os/osm2world/package.nix

```diff
@@ -0,0 +1,64 @@
+{ stdenv, lib, maven, fetchFromGitHub, makeWrapper
+, jogl, libXxf86vm, libGL, jre, jdk17 }:
+
+let
+  mavenJdk17 = maven.override {
+    jdk = jdk17;
+  };
+in mavenJdk17.buildMavenPackage {
+
+  pname = "osm2world";
+  version = "unstable-2024-02-25";
+
+  src = fetchFromGitHub {
+    owner = "tordanik";
+    repo = "OSM2World";
+    rev = "dda165854fb0b5b8c1116afed12703c4ee7f3fdb";
+    hash = "sha256-4mwoM3jCwkzlTkvAVJuIAsWgPGBWH25jmFz27oFl/E4=";
+  };
+
+  mvnHash = "sha256-Wcq3NOCschbv7e+1i4HyCn0BnRTbwF0V8yurME0QRXs=";
+  mvnParameters = "-Dmaven.test.skip"; # A single test requires internet connection.
```

Comment:
Can't we just skip that one test?

Reply:
@SuperSandro2000 , finally found a way. Please re-review. :)